src/dotnet-install.ps1

```diff
@@ -32,6 +32,8 @@
     - latest - most latest build on specific channel
     - 3-part version in a format A.B.C - represents specific version of build
           examples: 2.0.0-preview2-006120, 1.1.0
+.PARAMETER Internal
+    Set this switch to download internal builds. It is necessary to use FeedCredential option with this.
```

Comment:
nit: imo this wording is not common. Should we use "Applicable only when FeedCredential option is set"?

Reply:
Should it be put near FeedCredential?

Reply:
This parameter is quite similar to "channel", "version" and "quality" (as it also describes what we would like to install), so I put it near them. I can move "FeedCredential" near them also, though.

Reply:
I don't agree fully with the new wording either. 
```suggestion
    Download internal builds. Provide credentials via -FeedCredential parameter.
```

o

Reply:
Yes, I like your version more, @nohwnd, thank you!